src/util/usePanelParams.tsx

```diff
@@ -46,11 +66,23 @@ const usePanelParams = (): PanelHelper => {
     instance: params.get("instance"),
     profile: params.get("profile"),
     project: params.get("project") ?? "default",
+    identity: params.get("identity"),
+    group: params.get("group"),
+    idpGroup: params.get("idp-group"),
 
     clear: () => {
       clearParams();
     },
 
+    clearPanel: () => {
+      const newParams = new URLSearchParams(params);
+      newParams.delete("panel");
+      newParams.delete("identity");
+      newParams.delete("group");
+      newParams.delete("idp-group");
+      setParams(newParams);
+    },
+
```

Comment:
I think we don't need to distinguish `clear` and `clearPanel`. Can we merge the two functions and just have one?

We are missing to reset `instance`, `profile` and `project` params, maybe the logic can be added to the `clearParams` function?

Reply:
also left a comment detailing why only these params are deleted e.g. search params set prior to the panel opening should be kept.